src/pages/Home.vue

```diff
@@ -104,12 +105,12 @@ export default {
   },
   computed: {
     getDocumentPageLink() {
-      return "https://api.uat.fyndx1.de"
+      return this.moonlight_URL
         .replace("api", "partners")
         .concat(this.isApplicationLaunch ? DOC_APP_URL_PATH : DOC_URL_PATH);
     },
     isApplicationLaunch() {
```

Comment:
@jinalviranii Is this required for company level extension ?

Reply:
Yes required for both company and application level extension as per figma provided by deepu